routes/courses.js

```diff
@@ -1,96 +1,123 @@
+// courses.js
+
+// Specify strict mode for 'secure' code
```

Comment:
lets get rid of these 3. filename is kinda redundant & people should know what 'use_strict' does